modules/hosted-clusters-openstack-create-floating-ip.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hypershift-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hosted-clusters-openstack-create-floating-ip_{context}"]
+= Creating a floating IP for ingress
+
+If you want to make ingress available in a hosted cluster without manual intervention, you can create a floating IP address for it in advance.
+
+.Prerequisites
+
+* You have access to the {rh-openstack-first} cloud.
+
+.Procedure
+
+* Create a floating IP address by running the following command:
++
+[source,terminal]
+----
+$ openstack floating ip create <external_network_id>
+----
++
+--
+where:
+
+`<external_network_id>`:: Specifies the ID of the external network.
+--
+
+[NOTE]
```

Comment:
should we note `If you use a pre-defined floating IP address for ingress, you created a DNS record for the following wildcard`?

[0] https://github.com/openshift/openshift-docs/pull/93726/files#diff-4e88095fbf70caddfb8d482c362b8d48e21fd8e9769de153147f88e011d980deR19